src/Microsoft.ML.CpuMath/CpuMathUtils.netstandard.cs

```diff
@@ -6,8 +6,16 @@ namespace Microsoft.ML.Runtime.Internal.CpuMath
 {
     public static partial class CpuMathUtils
     {
+        // REVIEW NEEDED: AVX support cannot be checked in .NET Core App 2.0, so we assume Vector128 alignment for SSE.  Is it okay?
```

Comment:
Actually, we can check AVX support by calling into the native method that does this:

[https://github.com/dotnet/machinelearning/blob/94401d53b6bf41d8688103a7d5697ee251defd46/src/Native/CpuMathNative/Sse.cpp#L88-L105](https://github.com/dotnet/machinelearning/blob/94401d53b6bf41d8688103a7d5697ee251defd46/src/Native/CpuMathNative/Sse.cpp#L88-L105)

Reply:
I see - should I make the `.netstandard` implementation call some `AVX` intrinsics too since it can also check for AVX support, or should I leave out this comment and keep `GetVectorAlignment()` in this file `.netstandard.cs` to return only `Vector128Alignment`?

Reply:
Let's leave it the way you have it for now. But this is something that could be addressed in the future.